routes/web.php

```diff
@@ -27,9 +27,13 @@
 
     Route::get('/schedule', [ScheduleController::class, 'index'])->name('schedule.index');
     Route::get('/import', [ImportCourseController::class, 'index'])->name('import.index');
-    Route::get('/coursework', [CourseworkController::class, 'index'])->name('coursework.index');
 
     Route::get('/logout', [LoginController::class, 'logout'])->name('logout');
+
+    Route::get('/course',[CourseController::class,'index'])->name('course.index');
+    Route::get('/assignment',[AssignmentController::class,'index'])->name('assignment.index');
```

Comment:
These should be resource instead of get so that all of the routes become available